03/app.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import MenuItem from './MenuItem';
+
+const App = () => <MenuItem text="kontakt" url="/contact/"/>
+
+ReactDOM.render(
+    <ul>
+        <App/>
+    </ul>,
```

Comment:
To nie jest błąd, ale jakoś dziwnie wygląda. To `<App>` powinien zawierać `ul` oraz `<MenuItem>` :)